src/canvas/PatternPath.tsx

```diff
@@ -90,8 +93,8 @@ export class PatternPath implements IPatternPath {
         this._isPath2DValid = true;
         this._path2D.moveTo(firstPoint.getX(), firstPoint.getY());
 
-        const fittedCurve = CurveFitter.Fit(this._points);
-        this._path2D.quadraticCurveTo(fittedCurve.control.getX(), fittedCurve.control.getY(), fittedCurve.end.getX(), fittedCurve.end.getY());        
+        this._fittedCurve = CurveFitter.Fit(this._points);
```

Comment:
It doesn't seem right to me for the path to have a fitted curve, but I don't know what the solution to that is. Maybe it's the best choice for now.